Projects/AccountSDKIOSWebExample/AnotherSharedKeychainApp/ContentView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ContentView.swift
+//  AnotherSharedKeychainApp
+//
+//  Created by Daniel Echegaray on 2021-11-21.
+//
+
+import SwiftUI
+import AccountSDKIOSWeb
+
+struct ContentView: View {
+    static let config = ClientConfiguration(environment: .pre,
+                                            clientId:"App TWO ID",
+                                            redirectURI: URL(string: "com.sdk-example.pre.602504e1b41fa31789a95aa7:/login")!)
+                                            
+    static let client: Client =  Client(configuration: ContentView.config)
+
+    var body: some View {
+        Text("Hello, world!")
+            .padding()
+        
+        VStack(spacing: 50) {
```

Comment:
if you want to have spacing 50 between Text and second Vstack - you can put Text in first VStack. Now it looks a little unclear